tools/devtool

```diff
@@ -1180,6 +1180,12 @@ cmd_install() {
         esac
         shift
     done
+    
+    # Ensure install_path ends with a trailing slash
+    install_path="${install_path%/}/"
+
+    # Create the directory if it doesn't exist
+    mkdir -p "$install_path"
```

Comment:
Thanks for your contribution! Have you tried using `install`'s `-d` option? https://linux.die.net/man/1/install

Reply:
Looking at the man, ```-D -t``` seems the most appropriate, so that install_path is not interpreted as a file.

Reply:
Sounds good! Could you squash your commits into a single one?

Reply:
Done